src/link/link_info/bond.rs

```diff
@@ -199,6 +211,65 @@ impl std::fmt::Display for BondMode {
     }
 }
 
+#[derive(Debug, Clone, Copy, Eq, PartialEq, Default)]
+pub enum ArpValidate {
+    #[default]
+    None,
+    Active,
+    Backup,
+    All,
+    Filter,
+    FilterActive,
+    FilterBackup,
+    Other(u32),
+}
```

Comment:
@cathay4t 
In general, does it make sense to add the kernel documentation to these enums?

e.g. based on https://www.kernel.org/doc/html/v6.4-rc7/networking/bonding.html
```suggestion
/// Specifies whether or not ARP probes and replies should be validated in any mode that supports arp monitoring,
/// or whether non-ARP traffic should be filtered (disregarded) for link monitoring purposes.
#[derive(Debug, Clone, Copy, Eq, PartialEq, Default)]
pub enum ArpValidate {
    /// No validation or filtering is performed.
    #[default]
    None,
    /// Validation is performed only for the active slave.
    Active,
    /// Validation is performed only for backup slaves.
    Backup,
    /// Validation is performed for all slaves.
    All,
    /// Filtering is applied to all slaves. No validation is performed.
    Filter,
    /// Filtering is applied to all slaves, validation is performed only for the active slave
    FilterActive,
    /// Filtering is applied to all slaves, validation is performed only for backup slaves.
    FilterBackup,
    Other(u32),
}
```